examples/configs/neurobooth_os_config.json

```diff
@@ -1,26 +1,28 @@
 {
-    "data_out": "C:\\neurobooth\\neurobooth_data\\\\",
-    "nas": "Z:\\data\\\\",
+    "remote_data_dir": "Z:\\data\\\\",
```

Comment:
Do we need the quad backslash (which equates to \\ in the final string)? Can try to simplify while we're digging around in here.